irr/scripts/BindingGenerator.lua

```diff
@@ -317,20 +317,21 @@ local pointers = List();
 local loader = List();
 
 for s, str in ipairs( uniqueNames ) do
-	pointers:Add( ("\t%s %s = NULL;")( pfnFormat( str:upper() ), str ) );
```

Comment:
an explicit feature of the gl bindings was to merge all function calls between vendor prefixes, so that we don't have to write code like this
```c
if (GL.DoSomething)
    GL.DoSomething(12,3,4);
else if (GL.DoSomethingKHR)
    GL.DoSomethingKHR(12,3,4);
else if (GL.DoSomethingNV)
    GL.DoSomethingNV(12,3,4);
```

so I don't think changing this makes sense.

Reply:
The binding generator was reworked because of more findings.